equations.gms

```diff
@@ -403,14 +403,14 @@ QTotTransfOutput(runCy,EFS,YTIME)$TIME(YTIME)..
 * Compute transfers
 QTransfers(runCy,EFS,YTIME)$TIME(YTIME)..
          VTransfers(runCy,EFS,YTIME) =E=
-         (( (VTransfers(runCy,EFS,YTIME-1)*iResFeedTransfr(YTIME)*VFeCons(runCy,EFS,YTIME)/VFeCons(runCy,EFS,YTIME-1))$EFTOEFA(EFS,"LQD")+
+         (( (VTransfers(runCy,EFS,YTIME-1)*iResFeedTransfr(runCy,YTIME)*VFeCons(runCy,EFS,YTIME)/VFeCons(runCy,EFS,YTIME-1))$EFTOEFA(EFS,"LQD")+
           (
-                 VTransfers(runCy,"CRO",YTIME-1)*iResFeedTransfr(YTIME)*SUM(EFS2$EFTOEFA(EFS2,"LQD"),VTransfers(runCy,EFS2,YTIME))/
+                 VTransfers(runCy,"CRO",YTIME-1)*iResFeedTransfr(runCy,YTIME)*SUM(EFS2$EFTOEFA(EFS2,"LQD"),VTransfers(runCy,EFS2,YTIME))/
                  SUM(EFS2$EFTOEFA(EFS2,"LQD"),VTransfers(runCy,EFS2,YTIME-1)))$sameas(EFS,"CRO")   )$(iFeedTransfr(EFS,"2010"))$(NOT sameas("OLQ",EFS)) 
```

Comment:
`iFeedTransfr` has differentiation by country in MENA (`FEEDTRANS(CYall,EFS,YTIME)`), why was it removed in OPEN-PROM?

Reply:
It was removed because of a test. Now its fixed